apps/dapp/src/components/Pages/Core/Trade/use-swap-controller.tsx

```diff
@@ -92,9 +91,7 @@ export const useSwapController = () => {
 
   // Handles user input change
   const handleInputChange = async (value: string) => {
-    const bigValue = getBigNumberFromString(value || '0', getTokenInfo(state.inputToken).decimals);
-    const isZero = bigValue.eq(ZERO);
-    dispatch({ type: 'changeInputValue', value: isZero ? '' : value });
+    dispatch({ type: 'changeInputValue', value });
```

Comment:
It might be too much to add to this PR, but I think we should limit the max value you can manually type in to be capped at the _max balance_ of this token. As is it now, you can type in a higher amount than actually is in your wallet. Wdyt?

Reply:
pretty common to input an amount above your balance just to get the quote so I think we can leave this -- I do have a todo to display the error in the swap preview that the input is greater than your balance

Reply:
OK 👍